src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -487,25 +487,25 @@ void torque_driver_free_job(void *_job) {
     torque_job_free(job);
 }
 
-void *torque_driver_submit_job(void *_driver, const char *submit_cmd,
-                               int num_cpu, const char *run_path,
-                               const char *job_name) {
+void *torque_driver_submit_job(void *_driver, std::string submit_cmd,
+                               int num_cpu, fs::path run_path,
+                               std::string job_name) {
     auto driver = static_cast<torque_driver_type *>(_driver);
     torque_job_type *job = torque_job_alloc();
 
     logger->debug("Submitting job in: {}", run_path);
-    char *local_job_name = nullptr;
+    std::string local_job_name = "";
```

Comment:
This is implicit even on `std::string local_job_name;`

Reply:
Fixed